tests/TestCase/ORM/TableTest.php

```diff
@@ -2235,31 +2235,26 @@ public function testAfterSaveCommitWithNonAtomicAndTransactionRunning(): void
      */
     public function testAfterSaveNotCalled(): void
     {
-        /** @var \Cake\ORM\Table|\PHPUnit\Framework\MockObject\MockObject $table */
-        $table = $this->getMockBuilder(Table::class)
-            ->onlyMethods(['query'])
-            ->setConstructorArgs([['table' => 'users', 'connection' => $this->connection]])
-            ->getMock();
-        $query = $this->getMockBuilder('Cake\ORM\Query')
-            ->onlyMethods(['execute', 'addDefaultTypes'])
-            ->setConstructorArgs([$this->connection, $table])
```

Comment:
These tests started to fail. I felt this approach was pretty tightly coupled to several classes. I chose to only stub out the connection and sometimes the statement to get the same results.